scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SchedulerServiceImpl.java

```diff
@@ -1391,6 +1391,11 @@ private int executeVoidFunction(BiFunction<RestClient, Integer, Void> action, Li
 
     private String executeFunctionReturnStreamAsString(Function<RestClient, InputStream> function)
             throws ServiceException, RestServerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)